doc/api.rst

```diff
@@ -58,3 +58,19 @@ mne_bids.copyfiles
    copyfile_ctf
    copyfile_bti
    copyfile_kit
+
+mne_bids.sidecar_updates
+------------------------
+
+:py:mod:`mne_bids.sidecar_updates`:
+
+.. automodule:: mne_bids.sidecar_updates
+   :no-members:
+   :no-inherited-members:
+
+.. currentmodule:: mne_bids.sidecar_updates
+
+.. autosummary::
+   :toctree: generated/
+
+   update_sidecar_json
```

Comment:
don't add a full module for one function. Also 

from mne_bids.sidecar_updates import update_sidecar_json`

is very redundant. You repeat twice the same 2 words. I would do:

`from mne_bids import update_sidecar_json``